### PR TITLE
added new logo (330)

### PR DESCRIPTION
**Pull request by LeonKolyang**
Added a new logo: New Retro Wave
Also updated the music.html to include youtube links and the music.csv

### CODE REVIEW

**Comment by jbampton** [comment]
refs #598

File: docs/_data/tv.csv [hunk 1]
```diff
@@ -1,4 +1,5 @@
 name,image,title,designer
+The_Godfather,330,,
```
Comment (TJSteel) [inline docs/_data/tv.csv:2]: Should this not also be New_Retro_Wave_(band), or possibly the incorrect number has been assigned here?

The 2 images added in the music and tv .csv files are the same image, so I'd assume the names should match here

**Review by TJSteel — COMMENTED** [review]
Only one concern on the tv.csv file

**Review by jbampton — CHANGES_REQUESTED** [review]
We should not be editing the TV CSV file since we are only adding a Music logo.

File: docs/_data/tv.csv (hunk 1 above)
Comment (LeonKolyang) [reply docs/_data/tv.csv]: Sorry for that, I added the godfather logo before realising, that it already exists. I created a new commit, which deletes that row.

**Review by TJSteel — APPROVED** [review]
looks good to me :+1:

File: docs/pages/music.html [hunk 2]
```diff
@@ -13,6 +12,11 @@ <h3>{%- include jorge/jorge.html -%}</h3>
     {% else %}
         {% assign display = row.name | split: "_" | join: " " %}
     {% endif %}
+    {% if row.type == 'wiki' %}
+        {%  assign w = "https://en.wikipedia.org/wiki/" | append: row.name %}
+        {% else %}
```
Comment (jbampton) [inline docs/pages/music.html:17]: Indent missing

File: docs/pages/music.html [hunk 3]
```diff
@@ -13,6 +12,11 @@ <h3>{%- include jorge/jorge.html -%}</h3>
...
+        {% assign w = row.type %}
+        {% endif %}
```
Comment (jbampton) [inline docs/pages/music.html:19]: Indenting

**Review by jbampton — CHANGES_REQUESTED** [review]

**Comment by jbampton** [comment]
Needs a rebase

**Review by jbampton — CHANGES_REQUESTED** [review]
The links are broken for both the image title and also the image.
You have the wrong liquid markup.
You need to test your work locally.

**Comment by LeonKolyang** [comment]
Everything is running fine now, request [#619](https://github.com/slurpcode/slurp/pull/619) should be merged with this one I guess.

**Review by jbampton — CHANGES_REQUESTED** [review]
You need to follow the `games` example like I said and not introduce your own code.
We don't want two different ways of doing the same thing.
https://github.com/slurpcode/slurp/issues/598

**Comment by LeonKolyang** [comment]
Should meet the requirements now.

**Review by jbampton — APPROVED** [review]
💃

**Pull request merged**